docs/client/operations.md

```diff
@@ -0,0 +1,118 @@
+# <img align="center" src="../images/logo.png">  Calling Operations with Your Typescript Client
+
+AutoRest provides both synchronous and asynchronous method overloads for each service operation.
+Depending on your swagger definition, operations can be accessed through operation groups (TODO: link to swagger docs) on the client,
+or directly on the client.
+
+## Operation Group vs No Operation Group
+
+If your swagger defines an operation group for your operation (for example, in [this][operation_group_example] swagger, the operation `list`
+is part of operation group `application`), you would access the operation through `client.application.list()`.
+
+If there's no operation group, as in [this][mixin_example] case, you would access the operation directly from the client
+itself, i.e. `client.getDog()`.
+
+## Regular Operations
+
+### Sync Operations
+
+We don't generate sync operations, all of our operations are async.
+
+### Async Operations
+
+When calling our async operations, we use our async client, which is in a different module. Following the [example above](#sync-operations Sync Operations),
+our call to `getDog` looks like this:
+
+```js
+import { DefaultAzureCredential } from "@azure/identity";
+import { PetsClient } from "@azure/pets";
+
+let client: PetsClient;
+
+client = new PetsClient(new DefaultAzureCredential());
+const dog = await client.getDog();
+```
+
+## Long Running Operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation, followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has succeeded, to get the result.
+
+In concurrence with our [typescript guidelines][poller_guidelines], all of our long running operations are prefixed with `begin`, to signify the starting of the long running operation.
+
+For our example, we will use the long running operation generated from [this][example_swagger] swagger. Let's say we generated this swagger with package name `@azure/lro`.
+
+### Sync Long Running Operations
```

Comment:
Do we want to keep these Sync headers? It feels a bit like overkill to have them if we don't do Sync operations at all

Reply:
that's fair, I'll remove them. thanks!